src/Illuminate/Mail/TransportManager.php

```diff
@@ -97,8 +97,8 @@ protected function createMailDriver()
      */
     protected function createMailgunDriver()
     {
-        $client = new HttpClient;
         $config = $this->app['config']->get('services.mailgun', []);
+        $client = new HttpClient(Arr::get($config, 'guzzleOptions', []));
```

Comment:
I think, with that prefix "guzzle", you specify that this is guzzle-related options. No?